test_rmw_implementation/test/test_serialize_deserialize.cpp

```diff
@@ -141,6 +214,53 @@ TEST_F(CLASSNAME(TestSerializeDeserialize, RMW_IMPLEMENTATION), clean_round_trip
     rmw_get_error_string().str;
 }
 
+TEST_F(
+  CLASSNAME(
+    TestSerializeDeserialize,
+    RMW_IMPLEMENTATION), clean_round_trip_for_cpp_bounded_message) {
+  using test_message = test_msgs::msg::BoundedPlainSequences;
```

Comment:
@MiguelCompany nit: use camel case for C++ type names, as per [GSG](https://google.github.io/styleguide/cppguide.html#Type_Names).